Composer/packages/client/src/pages/botProject/create-publish-profile/ProfileFormDialog.tsx

```diff
@@ -70,6 +65,11 @@ export const ProfileFormDialog: React.FC<ProfileFormDialogProps> = (props) => {
     }
   };
 
+  const updateName = (e, newName) => {
+    setName(newName);
+    isValidateProfileName(newName);
```

Comment:
nit: name of the func -> ```validateProfileName```

Reply:
I'm just moving the function to be declared before it is used :)